### ISSUE TITLE
关于训练过程中参数文件缺失的问题

### ISSUE THREAD

**Issue by duola-wa** [lcshhh/TADPM#5]
作者您好，我在运行 train.sh 时遇到了错误，系统提示找不到 './data/param/3.pkl' 文件。从代码来看，这些 .pkl 文件似乎包含了牙齿变换参数的 ground truth。请问这些参数文件需要如何生成？感谢您的帮助

**Comment by danmoruchu** [comment]
> 作者您好，我在运行 train.sh 时遇到了错误，系统提示找不到 './data/param/3.pkl' 文件。从代码来看，这些 .pkl 文件似乎包含了牙齿变换参数的 ground truth。请问这些参数文件需要如何生成？感谢您的帮助

你好，你现在能够跑通了这个模型了吗

**Comment by liiad** [comment]
> 作者您好，我在运行 train.sh 时遇到了错误，系统提示找不到 './data/param/3.pkl' 文件。从代码来看，这些 .pkl 文件似乎包含了牙齿变换参数的 ground truth。请问这些参数文件需要如何生成？感谢您的帮助



> 作者您好，我在运行 train.sh 时遇到了错误，系统提示找不到 './data/param/3.pkl' 文件。从代码来看，这些 .pkl 文件似乎包含了牙齿变换参数的 ground truth。请问这些参数文件需要如何生成？感谢您的帮助

do you sovle the problem？

**Comment by lcshhh** [comment]
You're right, a ground truth file for the tooth transformation parameters is necessary. If the vertices of your pre- and post-orthodontic jaw models can be matched one-to-one, you can use the Kabsch algorithm to obtain an accurate rotation matrix. If not, you can use the ICP algorithm to obtain an approximate rotation matrix.
